addons/stock_delivery/wizard/choose_delivery_package.py

```diff
@@ -22,8 +23,10 @@ def _compute_weight_uom_name(self):
     @api.depends('delivery_package_type_id')
     def _compute_shipping_weight(self):
         for rec in self:
-            move_line_ids = rec.picking_id._package_move_lines(
-                batch_pack=self.env.context.get('batch_pack')
+            move_line_ids = rec.move_line_ids.filtered(lambda m:
```

Comment:
Why can't you use _package_move_lines? Seems strange since we want to get the weight for the future package